index.js

```diff
@@ -1 +1,116 @@
-// implement your API here
+const express = require("express");
+const cors = require("cors");
+
+const Data = require("./data/db.js");
+
+const server = express();
+
+server.use(express.json());
+server.use(cors());
+
+//C-R-U-D
+
+//Read - display the array of users
+server.get("/api/users", (req, res) => {
+  Data.find()
+    .then(user => {
+      res.status(200).json(user);
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ error: "The user's information could not be retrieved" });
```

Comment:
You can keep it on one line.

`res.status(500).json({...})`

Reply:
I tried so hard. I think prettier did that auto formatting